### PR TITLE
Feature/category

### PR DESCRIPTION
**Pull request by mluena**
> **IMPORTANT: ready to merge when categories get updated in CARTO**

This PR:

- Includes function that fetches categories from CARTO.
- Gets rid of category pages to load them dynamically from country Page through CardInfo component.
- Add Spinner component.
- Gets rid of TabsInfo constants.
- Reorganize styles
- Gets rid of categories.js

## Testing instructions

Go to country page and navigate through tabs

### CODE REVIEW

File: src/components/spinner/styles.scss [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@import 'styles/settings';
```
Comment (davidsingal) [inline src/components/spinner/styles.scss:1]: This is not needed anymore as we are using global styles.

File: src/pages/country/component.js [hunk 2]
```diff
@@ -3,19 +3,35 @@ import PropTypes from 'prop-types';
...
+import { fetchCategories } from 'components/widget/utils';
+import useAxios from 'axios-hooks';
 
 const CountryPage = ({ iso, current }) => {
-  const infoPage = TabsInfo.find((info) => info.category === current);
+  const [{ data, loading }] = useAxios(fetchCategories());
+  const tabs = data && data.rows ? data.rows : null;
+
+  const currentTab = tabs ? tabs.find((tab) => tab.slug === current) : null;
+  const name = currentTab ? currentTab.name : null;
```
Comment (davidsingal) [inline src/pages/country/component.js:18]: const { name, slug } = currentTab && {};

File: src/pages/country/component.js [hunk 3]
```diff
@@ -3,19 +3,35 @@ import PropTypes from 'prop-types';
...
+
   return (
     <div className="l-country">
       <Header />
       <Hero iso={iso} />
-      <div className="country-content">
-        <Navigation tabs={TabsInfo} iso={iso} currentTab={current} />
-        {infoPage.content}
-      </div>
+      {loading ? (
```
Comment (davidsingal) [inline src/pages/country/component.js:25]: It is very important to check loading and data, it is not enough to check only loading.

File: src/pages/country/component.js (hunk 2 above)
Comment (mluena) [reply src/pages/country/component.js:18]: Screenshot 2020-04-21 at 08 52 11

**Pull request merged**